csharp/src/Ice/SlicSocket.cs

```diff
@@ -23,8 +23,9 @@ public override TimeSpan IdleTimeout
 
         internal int BidirectionalStreamCount;
         internal AsyncSemaphore? BidirectionalStreamSemaphore;
+        internal int MaxBidirectionalStreams;
+        internal int MaxUnidirectionalStreams;
         internal readonly object Mutex = new();
```

Comment:
Not directly related to this PR: it's desirable for mutexes and other synchronization objects to be private, not shared between several classes.

Reply:
I'll fix this in another PR.